dask/dataframe/tests/test_utils_dataframe.py

```diff
@@ -126,13 +126,19 @@ def __next__(self):
     assert len(meta.index) == 0
 
     # Categoricals
-    meta = make_meta({"a": "category"}, parent_meta=df)
+    meta = make_meta({"a": "category"}, parent_meta=df)  # fails
```

Comment:
@jsignell Thanks for [suggesting a solution](https://github.com/dask/dask/issues/9291#issuecomment-1202684720)!

Looks like the change fixes #9291 but breaks this line, with:
```
TypeError: Index(...) must be called with a collection of some kind, None was passed
```
Any idea what's happening?